webdriver/bidi/webExtension.json

```diff
@@ -37,14 +37,14 @@
             "spec_url": "https://w3c.github.io/webdriver-bidi/#command-webExtension-install",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": "135",
+                "partial_implementation": true,
+                "notes": "Basic support"
```

Comment:
```suggestion
                "notes": "Basic support (only `type: 'path'` is supported)"
```

Reply:
Thanks, I have added the following note to the version cell in the spreadsheet:

```
Only `type: "path"` is supported, not `type: "archivePath"` or `type: "base64"`.
```